backend/Services/BedService.cs

```diff
@@ -0,0 +1,67 @@
+namespace backend.Services;
+
+using DTOs;
+using Interfaces;
+using System.Collections.Generic;
+using System.Linq;
+
+public class BedService : IBedService
+{
+    private List<BedDTO> _beds = new List<BedDTO>
+    {
+        new BedDTO
+        {
+            BedID = Guid.NewGuid(),
+            BedQuantity = 2,
+            Capacity = "2",
+            Size = "king"
+        },
+        new BedDTO
+        {
+            BedID = Guid.NewGuid(),
+            BedQuantity = 1,
+            Capacity = "1",
+            Size = "single"
+        }
+    };
+
+    public List<BedDTO> GetBeds()
+    {
+        return _beds;
+    }
+
+    public BedDTO GetBedById(Guid bedID)
+    {
+        return _beds.FirstOrDefault(b => b.BedID == bedID);
+    }
+
+    public bool CreateBed(BedDTO bedDto)
+    {
+        _beds.Add(bedDto);
```

Comment:
create method on services should assure the correct addition of an element

Reply:
Done